multi-cluster-orchestrator/samples/hello_world/README.md

```diff
@@ -0,0 +1,420 @@
+# Hello World Sample
+
+In this quickstart, you learn how to deploy a Hello World sample as a
+multi-region [Google Kubernetes Engine (GKE)][0] workload using
+[Multi-cluster Orchestrator][18] and [Multi-cluster Gateway][19].
+
+Infrastructure as Code (IaC) is a practice of managing and provisioning software
+infrastructure resources using code. Terraform is a popular open source IaC tool
+that supports a wide range of Cloud services, including GKE. As a GKE platform
+administrator, you can use Terraform to standardize configuration of your
+Kubernetes clusters and streamline your DevOps workflows. To learn more, see
+[Terraform support for GKE][1].
+
+## Objectives
+
+- Deploy the pre-requisite infrastructure, including:
+  - Hub GKE cluster running Multi-cluster Orchestrator and Argo CD
+  - Worker GKE clusters in multiple-regions
+- Deploy the Hello World sample workload using Multi-cluster Orchestrator and
+  Argo CD
+- Load test the Hello World server using Multi-cluster Gateway to trigger multi-cluster scaling
+
+## Before you begin
+
+Take the following steps to enable the Kubernetes Engine API:
+
+  1. In the Google Cloud console, on the project selector page, select or [create a
+  Google Cloud project][2].
+
+  > [!TIP]
+  > If you don't plan to keep the resources that you create in this procedure,
```

Comment:
TF kinda makes this unnecessary, they just need to remember to `tf destroy`?

Reply:
Most resource, however it's not really greenfield, for example APIs are still enabled.

Reply:
I'll drop as now that we are using the `default` network it's much less of an issue.